### DISCUSSION TITLE
Setting storage engine in schema file

### DISCUSSION THREAD

**Discussion by jtubbenhauer** [prisma/prisma#14734] [Q&A] [answered]
I'm using a MariaDB (10.3.35) database on my shared hosting, which for whatever reason defaults to MyISAM for new tables. I'm hoping to find a way to specify the engine within the Prisma schema file.

My current workaround is to create a migration with '--create-only', then manually add 'ENGINE INNODB' to each table.

I might be missing something obvious here as I'm new to MySQL/MariaDB.

**Comment by austincrim** [comment, marked as answer]
Hey @jtubbenhauer 👋,

Your current workaround is the solution, as far as I know. We note this specifically [in our docs](https://www.prisma.io/docs/concepts/database-connectors/mysql#engine).

If you have an idea on how we could solve this, feel free to open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!

**Comment by IArnaut2** [comment]
This is absolutely idiotic. Why such feature does not exist?
